src/app/list-company/list-company.component.html

```diff
@@ -1,3 +1,27 @@
-<p>
-  list-company works!
-</p>
+<mat-form-field>
```

Comment:
Add a container div so the filter input and the list don't take all the place available